program/CORE-V IDE prelminary project proposal.md

```diff
@@ -0,0 +1,251 @@
+- Build and Delivery Description
+
+
+
+
+## Virtual customer
+The project team would like to expore the concept of virtual customer to help provide suggestions on feature requirements, participate in ongoing backlog grooming, and review and feedback on CORE-V IDE as it is released. It is suggested that CMC (Hugh Politt-Smith) can act in this role as the IDE is used with the FPGA implementation of CV32E40P
+
+
+## Summary of requirements
+
+The present document gives a present snapshot of requirements and Preliminary Project Proposal, the requirements document https://github.com/openhwgroup/core-v-sw/blob/master/planning/openhw-ide-requirement.md will be used as the reference going forward. 
+
+### Introduction
+Use case list would be useful
+
+### Initial project requirements 
+
+1. Support for 64 bit Windows only
+2. Supplied as a ZIPped archive...no installer needed...just extract and ready to run
+3. �OpenHW IDE� splash-screen and "about" box with openHW Group graphics. 
+4. Set version to v0.0.1
+5. Build on latest, released Eclipse, CDT and GNU MCU  plug-ins (now known as Eclipse Embedded CDT (C/C++ Development Tools))
+6. Latest Embecosm CORE-V toolchain
+7. OpenOCD (Open On-Chip Debugger, with JTAG support) for CORE-V
+8. Simple Eclipse project ready-to-run on Genesys 2 board (e.g. LED toggle) including Opella-LD OpenOCD launch (Opella-LD is 	Ashling's low-cost, FTDI based probe not to be confused with the existing Opella-XD)
+9. OpenHW IDE getting started guide documentation with Genesys 2 board 
+
+
+### Future enhancements:
+
+10. Adding Linux support
+11. Automating the build to �pull� all the latest components from their respective repos etc
+12. Improved documentation and more board examples
+13. Various improvements
+
+
+
+## Explanation of why OpenHW should do this project
+
+The industry will require a Embedded CDT IDE tuned to CORE-V. 
+- OpenHW is developing the CORE-V RTL, FPGA instantiations of the CORE-V RTL, and the s/w toolchain to support CORE-V.
+- OpenHW has a close parternship with the Eclipse Foundation. 
+- Building the CDT enhancements for CORE-V builds on the strengths of the OpenHW ecosystem. 
+
+For the above reasons, OpenHW is best placed to do this needed work.
+
+## Industry landscape: description of competing, alternative, or related efforts in the industry
+
+### Related efforts to be described
+
+- Eclipse CDT 
+- (GNU MCU) Eclipse Embedded CDT (Describe support for existing RISC-V devices)
+- Eclipse Theia
+
+## External dependencies
+ 
+The first build of CORE-V CDT builds on the following:
+- Eclipse Platform 2020-09
+- Eclipse CDT 10.0
+- Other components from Eclipse SimRel 2020-09
+- A suitable tool chain (see options below)
+
+The CORE-V IDE will be capable of integration with the following tool chain options. As per the project requirements above, the main focus of the CORE-V IDE will be the OpenHW CORE-V reference toolchain.

```

Comment:
More correctly:

----

The CORE-V IDE will be capable of integrating with any compliant tool chain, notably GNU and Clang/LLVM.  A the time of writing, the following tool chains are available.